articles/postgresql/concepts-hyperscale-high-availability.md

```diff
@@ -17,6 +17,11 @@ incoming connections from the failed node to its standby. Failover happens
 within a few minutes, and promoted nodes always have fresh data through
 PostgreSQL synchronous streaming replication.
 
+Without HA enabled each Hyperscale (Citus) node has its own LRS storage with three (3) synchronous replicas maintained by Azure Storage service. 
```

Comment:
Thanks for the extra details. Here's another way to phrase what you said which conforms better with the Acrolinx style guidelines:

```diff
@@ -6,7 +6,7 @@ ms.author: jonels
 ms.service: postgresql
 ms.subservice: hyperscale-citus
 ms.topic: conceptual
-ms.date: 11/04/2019
+ms.date: 9/24/2021
 ---

 # High availability in Azure Database for PostgreSQL – Hyperscale (Citus)
@@ -17,6 +17,19 @@ incoming connections from the failed node to its standby. Failover happens
 within a few minutes, and promoted nodes always have fresh data through
 PostgreSQL synchronous streaming replication.

+Even without HA enabled, each Hyperscale (Citus) node has its own locally
+redundant storage (LRS) with three synchronous replicas maintained by Azure
+Storage service.  If there's a single replica failure, it’s detected by Azure
+Storage service and is transparently re-created. For LRS storage durability,
+see metrics [on this
+page](/azure/storage/common/storage-redundancy.md#summary-of-redundancy-options).
+
+When HA *is* enabled, Hyperscale (Citus) runs one standby node for each primary
+node in the server group. The primary and its standby use synchronous PostgreSQL
+replication. This replication allows customers to have predictable downtime if
+a primary node fails. In a nutshell, our service detects a failure on primary
+nodes, and fails over to standby nodes with zero data loss.
+
 To take advantage of HA on the coordinator node, database applications need to
 detect and retry dropped connections and failed transactions. The newly
 promoted coordinator will be accessible with the same connection string.
```

I also made the link locale-agnostic and bumped the date.

Does this change still match what you're trying to say?

Reply:
@kruti-m over to you.